.github/workflows/unit_test.yml

```diff
@@ -0,0 +1,24 @@
+name: Unit_test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@ec106b438a1ff6ff109590de34ddc62c540232e0
```

Comment:
`uses: ruby/setup-ruby@v1`
Is better, since not specific commit, but specific verison